### PR TITLE
Create localmonero.co.xml

### PR DESCRIPTION
**Pull request by LocalMonero**

### CODE REVIEW

**Comment by pipboy96** [comment]
@LocalMonero Thanks for contribution! Since you are the website owner, it's recommended that you use [HSTS preloading](https://hstspreload.org/) instead. It's more effective since it doesn't require the user to have HTTPS Everywhere installed, as long as they are on a modern version of Chrome or Firefox.

**Comment by LocalMonero** [comment]
@pipboy96  We'd like to but there are issues with backwards compatibility related to it. Our main reason to add this to HTTPS Everywhere is to protect people using Tor browser (which has HTTPS Everywhere installed) against malicious exit nodes. Same goes for #19457. We request you merge both of these as soon as possible.

File: src/chrome/content/rules/localmonero.co.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<ruleset name="localmonero.co">
+	<target host="localmonero.co" />
+	<target host="meet.localmonero.co" />
+  <target host="forums.localmonero.co" />
+
+	<rule from="^http:" to="https:" />
+</ruleset>
```
Comment (pipboy96) [inline src/chrome/content/rules/localmonero.co.xml:7]:
```suggestion
<ruleset name="localmonero.co">
	<target host="localmonero.co" />
	<target host="www.localmonero.co" />
	<target host="meet.localmonero.co" />
	<target host="mta-sts.agoradesk.com" />
	<target host="forums.localmonero.co" />

	<rule from="^http:" to="https:" />
</ruleset>
```

Comment (LocalMonero) [reply src/chrome/content/rules/localmonero.co.xml:7]: You seem to have accidentally added "mta-sts.agoradesk.com" to the LocalMonero rule.

Comment (pipboy96) [reply src/chrome/content/rules/localmonero.co.xml:7]: @LocalMonero Thanks for noticing!

Comment (LocalMonero) [reply src/chrome/content/rules/localmonero.co.xml:7]: mta-sts.localmonero.co isn't really needed, though.

Comment (pipboy96) [reply src/chrome/content/rules/localmonero.co.xml:7]: @LocalMonero mta-sts.localmonero.co is removed.

Comment (pipboy96) [reply src/chrome/content/rules/localmonero.co.xml:7]: @LocalMonero Are these changes OK? If yes, please press "Commit suggestion", and after that I will request review from another maintainer (I can't merge PRs by myself).

Comment (LocalMonero) [reply src/chrome/content/rules/localmonero.co.xml]: @pipboy96 Done.

**Comment by pipboy96** [comment]
@LocalMonero Thanks for contribution!

**Pull request merged**